QModManager/Patching/Patcher.cs

```diff
@@ -65,7 +65,8 @@ internal static void Patch()
 
                 try
                 {
-                    Logger.Info($"Folder structure:{IOUtilities.GetFolderStructureAsTree()}");
+                    Logger.Info($"Folder structure:");
```

Comment:
The string interpolation is not needed here, though I'm just being fussy as I'm not sure if it's gonna waste much performance, probably the compiler is smart enough to just ignore it.

```suggestion
                    Logger.Info("Folder structure:");
```